pip_review/__main__.py

```diff
@@ -244,55 +136,59 @@ def confirm(question):
     return answer == 'y'
 
 
+def get_outdated_packages(local=False, pre_release=False, user=False):
+    command = ['pip', 'list', '--outdated', '--disable-pip-version-check']
```

Comment:
I know why `--disable-pip-version-check` is breaking the build: it works wonderfully well for recent versions of pip, but at the end of the test we are force-downgrading to pip<6.0 (so 1.5.x), which apparently doesn't have the option. So we need some other solution to get around the pip update warning for old versions of pip.